src/lib/routes/admin-api/events.test.ts

```diff
@@ -116,11 +116,11 @@ test('should anonymise any PII fields, no matter the depth', async () => {
         new ProjectAccessAddedEvent({
             createdBy: 'some@email.com',
             data: {
-                groups: [
+                roles: [
                     {
-                        name: 'test',
-                        project: 'default',
-                        users: [{ username: testUsername }],
+                        roleId: 1,
+                        groupIds: [1, 2],
+                        users: [{ id: 1, username: testUsername }],
```

Comment:
should we drop the username? Feels like a user id should be enough?

Reply:
This is only in the test, it's actually a PII test that just happens to use that particular event.

Reply:
I can take a look at removing this particular test/making it a more appropriate test.

Reply:
I've added a comment to the test that explains the use case and that it's specific to testing PII and not directly relevant of the event itself